vector/src/main/java/im/vector/app/features/roomprofile/polls/RoomPollsPagerAdapter.kt

```diff
@@ -19,15 +19,19 @@ package im.vector.app.features.roomprofile.polls
 import androidx.fragment.app.Fragment
 import androidx.viewpager2.adapter.FragmentStateAdapter
 import im.vector.app.features.roomprofile.polls.active.RoomActivePollsFragment
+import im.vector.app.features.roomprofile.polls.ended.RoomEndedPollsFragment
 
 class RoomPollsPagerAdapter(
         private val fragment: Fragment
 ) : FragmentStateAdapter(fragment) {
 
-    override fun getItemCount() = 1
+    override fun getItemCount() = 2
 
     override fun createFragment(position: Int): Fragment {
-        return instantiateFragment(RoomActivePollsFragment::class.java.name)
+        return when (position) {
+            0 -> instantiateFragment(RoomActivePollsFragment::class.java.name)
+            else -> instantiateFragment(RoomEndedPollsFragment::class.java.name)
```

Comment:
I would prefer having an explicite list of cases with 0, 1 and trigger an error in the else branch which should never happen, wdyt?

Reply:
Yes I decided to throw an exception in that case: see [85cfa43](https://github.com/vector-im/element-android/pull/7886/commits/85cfa433d9e15362c0b907d19e253c01cff665c5)